pkg/controllers/status/cluster_status_controller.go

```diff
@@ -224,6 +226,8 @@ func (c *ClusterStatusController) syncClusterStatus(cluster *clusterv1alpha1.Clu
 			}
 			currentClusterStatus.NodeSummary = getNodeSummary(nodes)
 			currentClusterStatus.ResourceSummary = getResourceSummary(nodes, pods)
+
+			setAllocatableModelings(cluster, nodes, pods, &currentClusterStatus)
```

Comment:
```suggestion

			if features.FeatureGate.Enabled(features.CustomizedClusterResourceModeling) {
				currentClusterStatus.ResourceSummary.AllocatableModelings = xxxx
			}
```
Why not keep this format??

Reply:
Mainly because now syncClusterStatus is complex enough that adding an if judgment will cause the cyclomatic complexity to exceed the standard. Can we skip check for code readness?